### PR TITLE
Add path option to cli

### LINKED ISSUE #1507: Run feast cli command without cd into feature repo directory
**Is your feature request related to a problem? Please describe.**
Is it possible to run a feast command i.e. feast apply, without cd into the feature repo directory?

**Describe the solution you'd like**
feast -p feature_repo apply
feast -p feature_repo materialize-incremental $CURRENT_TIME

**Possible solution**
PR #1509

### PR DESCRIPTION
**Pull request by tedhtchang**
Signed-off-by: ted chang <htchang@us.ibm.com>

**What this PR does / why we need it**:
Specify a feature repo path using `feat --chdir path-to-a-feature-repository  COMMAND ..`
**Which issue(s) this PR fixes**:
Possibly Fixes #1507

**Does this PR introduce a user-facing change?**:
yes
```release-note
The --chdir or -c option allows users to run a feast command outside of a feature repo directory. 
```

### CODE REVIEW

File: sdk/python/feast/cli.py [hunk 1]
```diff
@@ -38,14 +38,18 @@
 
 
 @click.group()
-def cli():
+@click.option("--path", "-p", help="Feature repo path")
```
Comment (woop) [inline sdk/python/feast/cli.py:41]:
```suggestion
@click.option("--path", "-p", help="Path to feature repository")
```

**Comment by woop** [comment]
Thanks @tedhtchang, can we maybe get a single test to confirm that this functionality works?

**Comment by tedhtchang** [comment]
@tsotnet How do I find out what failed in the `linter / lint-python` check ?

**Comment by tedhtchang** [comment]
> Thanks @tedhtchang, can we maybe get a single test to confirm that this functionality works?
@woop Sure. I added a test.

**Comment by woop** [comment]
> @tsotnet How do I find out what failed in the `linter / lint-python` check ? Never mind the lint-python check showed up now.

For future reference, `make lint` and `make format` can be used to check linting locally and reformat the code.

**Pull request merged**